src/Components/ReviewCards.jsx

```diff
@@ -1,4 +1,12 @@
+import { useNavigate } from "react-router-dom";
+
+
 export default function ReviewCards({ reviews }) {
+  const navigate = useNavigate()
+  const handleOnClick = (id) => {
```

Comment:
The only thing negative about this box is it doesn't enhance the user experience, the user is very unlikely to know the id number and if they did they could just add it to the URL